MAINTAINERS.md

```diff
@@ -0,0 +1,92 @@
+# Maintainers Manual
+
+Welcome, Maintainers!
+
+Just like the core Astro code base, and other code repositories within the Astro organization, you also have extra permissions within the Docs repo, too.
+
+When you do find yourself wanting or needing to get involved in the Docs, here is some handy guidance for helping to maintain this repository.
+
+**"How can I help, if I have some time today?"**
+
+This is a recurring theme in the #docs Discord channel. Maintainers regularly post messages to `@team-docs` answering _everyone's obvious burning question_, with specific or general prompts and nudges.
+
+The following is the answer to that question **for Astro Maintainers**, so that you can all feel comfortable jumping in and helping out.
+
+
+## Triaging Issues and PRs
+
+Docs uses a label system in GitHub to categorize Issues and PRs **according to general topic area**.
+
+The docs site involves not only content (writing and technical content), but also everything involved in managing an Astro site: site UI, architecture, accessibility (a11y) issues and internationalization (i18n) with multiple languages in active translation.
+
+When you see PRs that have not yet been labelled, you can help by adding one or more labels. Our common labels include:
+
+- `i18n` includes page translations and other efforts to keep the translators organized, including GitHub actions, automation, and site UI relevant to supporting multiple languages.
+
+- `add new content` is used for something previously undocumented on the website. (i.e. We haven't thought in details about how to write about this topic before.) This tag is useful because it reminds us that we should make sure we really understand this new thing, and any implications elsewhere in the Docs, while documenting it. (Does it replace an existing feature that we need to deprecate? Does this change an instruction or recommendation documented elsewhere? If a brand new feature, does it require a technical documentation in Reference and also a more user-friendly mention somewhere in Learn?)
+
+- `improve existing documentation` is used for smaller, but non-trivial documentation improvements to items that are currently documented. This distinction is helpful because we have already considered this feature as part of the entire Docs site, and the change might be small enough to be localized. For example, improving the wording of an instruction or adding an extra example to illustrate a particular nuance of a topic that we have already documented.
+
+- `site improvement` is a broad category, but helps us distinguish content and writing tasks from site issues and maintanance. This distinction is helpful to allow community members and maintainers to easily filter tasks to either focus on or ignore content writing Issues and PRs. This allows community members (and maintainers!) more easily contribute according to their own skills and interests.
```

Comment:
Although it could be inferred, this wasn’t 100% clear which of the two things (site issues vs content) this label was for. Maybe reword like this?

```suggestion
- `site improvement` is a broad category, but is used to label site issues and maintenance tasks as opposed to content and writing tasks. This distinction is helpful to allow community members and maintainers to easily filter tasks to either focus on or ignore content writing Issues and PRs. This allows community members (and maintainers!) more easily contribute according to their own skills and interests.
```